lib/net/common.js

```diff
@@ -91,6 +91,26 @@ exports.MAX_MESSAGE = 8 * 1000 * 1000;
 
 exports.BAN_TIME = 24 * 60 * 60;
 
+/**
+ * Amount of time after which to reset DOS peers.
+ * @const {Number}
+ * @default
+ */
+
+exports.DOS_RESET_TIME = 30 * 60;
+
+/*
+ * Precomputed decay factor for first 64 seconds.
+ * @const {Object}
+ * @default
+ */
+
+const dosDecayFactor = {};
```

Comment:
I benchmarked an array vs an object as well, the object was faster :+1:

Reply:
Cool :+1: